pkg/kube/apis/extendedstatefulset/v1alpha1/types.go

```diff
@@ -27,6 +27,8 @@ var (
 	LabelAZName = fmt.Sprintf("%s/az-name", apis.GroupName)
 	// LabelPodOrdinal is the index of pod ordinal
 	LabelPodOrdinal = fmt.Sprintf("%s/pod-ordinal", apis.GroupName)
+	// LabelEStsName is the name of the ExtendedStatefulSet owns this resource
```

Comment:
@rohitsakala 
I think this label will label pods owned by the child statefulSet as expected. 
And it can label statefulSets if you will, but it's duplicated as we use `ownerReferences` to track extendedStatefulSet with its statefulSet.

Reply:
Ahh..okies.